### PR TITLE
move installation scripts to npm package

### PR DESCRIPTION
**Pull request by thfield**
directions in README.md
INSTALL scripts moved to https://github.com/pointblue/application-scripts

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,7 +2,11 @@
   "private": true,
   "scripts": {
     "prod": "gulp --production",
-    "dev": "gulp watch"
+    "dev": "gulp watch",
+    "build": "./node_modules/application-scripts/BUILD",
+    "deploy": "./node_modules/application-scripts/DEPLOY",
+    "inst": "./node_modules/application-scripts/INSTALL",
```
Comment (stereocilia) [inline package.json]: can we change this to `setup` or `install-app` or something? `inst` makes me think of `instrument`

File: README.md [hunk 2]
```diff
@@ -22,22 +22,23 @@ From the `devops` project you cloned - `devops/devenv/apps/downloader`
 ### Install Steps 
  
 1. clone repo to the install path (noted above)
-2. run `./INSTALL` 
+1. run `npm install --no-bin-links`
```
Comment (stereocilia) [inline README.md]: maybe:
run `npm install --no-bin-links` or `npm install` if your host machine is linux
no idea why this fails for me

**Comment by stereocilia** [comment]
Here's the INSTALL script that worked for me:
```
mkdir -p storage/framework/sessions
mkdir -p storage/framework/views
mkdir -p storage/framework/cache
mkdir -p bootstrap/cache
composer install
cp ./.env.example ./.env
php artisan key:generate
./node_modules/application-scripts/BUILD
```

**Comment by bsechelski-pointblue** [comment]
The revised install with full path worked for me too. I'm wondering if there's any way to use an env variable vs './node_modules/application-scripts/'

**Comment by stereocilia** [comment]
perhaps [https://www.npmjs.com/package/get-installed-path](https://www.npmjs.com/package/get-installed-path) ?

**Comment by thfield** [comment]
@stereocilia were you running `npm install`/`npm install --no-bin-links` in your vagrant or in your host?

File: README.md (hunk 2 above)
Comment (thfield) [inline README.md]: just realized this is the right place to ask my earlier question: are you running `npm install` in your host or guest machine?

File: package.json (hunk 1 above)
Comment (thfield) [inline package.json]: easy to change

File: README.md [hunk 3]
```diff
@@ -22,22 +22,23 @@ From the `devops` project you cloned - `devops/devenv/apps/downloader`
...
+1. run `npm run inst`
```
Comment (thfield) [inline README.md]: do you remember which dependencies?

**Pull request merged**